.github/workflows/ci.yml

```diff
@@ -1144,7 +1144,7 @@ jobs:
         include:
           - package_name: '@tryghost/admin-x-activitypub'
             package_path: 'apps/admin-x-activitypub'
-            cdn_paths: 'https://cdn.jsdelivr.net/ghost/admin-x-activitypub@0/dist/admin-x-activitypub.js'
+            cdn_paths: 'https://cdn.jsdelivr.net/ghost/admin-x-activitypub@1/dist/admin-x-activitypub.js'
```

Comment:
Nice - maybe it would be good if we can just clear the cache of everything so we don't have to remember to bump this - but not sure if we can

Reply:
But that should be enough right? if we go from 1 to 2, we don't need to be purging 1 anymore as all changes going forward will be on 2?

Reply:
Discussed with @allouis on a call - We have added a new `CURRENT_MAJOR` placeholder that works in the same way as the `CURRENT_MINOR` placeholder. This will allow us to always reference the latest MAJOR version to clear the cache for (which is what we want). My suggestion for using `@latest` would be the literal latest version, not just the major